kodilla-stream/src/test/java/com/kodilla/stream/portfolio/BoardTestSuite.java

```diff
@@ -135,4 +140,23 @@ public void testAddTaskListFindLongTasks() {
         //Then
         Assert.assertEquals(2, longTasks);
     }
+
+    @Test
+    public void testAddTaskListAverageWorkingOnTask() {
+        //Given
+        Board project = prepareTestData();
+
+        //When
+        List<TaskList> inProgressTasks = new ArrayList<>();
+        inProgressTasks.add(new TaskList("In progress"));
+        double averageWorkingOnTask = project.getTaskLists().stream()
+                .filter(inProgressTasks::contains)
+                .flatMap(n -> n.getTasks().stream())
```

Comment:
Co to jest `n`? Spróbuj uzyć bardziej deskryptywnej nazwy,

Reply:
ok